lib/v2/datasource/local/qr_code_service.dart

```diff
@@ -12,59 +11,10 @@ class QrCodeService {
       return Future.value(ErrorResult('Invalid QR Code'));
     }
 
-    final _SeedsESR esr = _SeedsESR(uri: scanResult);
```

Comment:
There was s custom, duplicate _SeedsESR class around - not sure why. Unified them into SeedsESR.